galasa-managers-parent/galasa-managers-workflow-parent/dev.galasa.githubissue.manager/src/main/java/dev/galasa/githubissue/internal/GitHubIssueManagerImpl.java

```diff
@@ -3,61 +3,288 @@
  */
 package dev.galasa.githubissue.internal;
 
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Method;
+import java.net.URISyntaxException;
 import java.util.List;
 
 import javax.validation.constraints.NotNull;
 
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
+import org.osgi.service.component.annotations.Component;
+
+import com.google.gson.JsonObject;
 
 import dev.galasa.ManagerException;
+import dev.galasa.Test;
 import dev.galasa.framework.spi.AbstractManager;
+import dev.galasa.framework.spi.ConfigurationPropertyStoreException;
 import dev.galasa.framework.spi.IFramework;
 import dev.galasa.framework.spi.IManager;
+import dev.galasa.framework.spi.Result;
+import dev.galasa.framework.spi.language.GalasaMethod;
 import dev.galasa.framework.spi.language.GalasaTest;
 import dev.galasa.githubissue.GitHubIssue;
+import dev.galasa.githubissue.GitHubIssueManagerException;
+import dev.galasa.githubissue.Issue;
+import dev.galasa.githubissue.internal.properties.GitHubIssueInstanceUrl;
+import dev.galasa.githubissue.internal.properties.GitHubIssuePropertiesSingleton;
+import dev.galasa.http.HttpClientException;
+import dev.galasa.http.HttpClientResponse;
+import dev.galasa.http.IHttpClient;
+import dev.galasa.http.IHttpManager;
+import dev.galasa.http.spi.IHttpManagerSpi;
 
+@Component(service = { IManager.class })
 public class GitHubIssueManagerImpl extends AbstractManager {
 	
 	private static final Log logger = LogFactory.getLog(GitHubIssueManagerImpl.class);
 	
+	protected static final String NAMESPACE = "githubissue";
+	
 	private boolean required;
 	
+	private GalasaTest galasaTest;
+	
 	private GitHubIssue gitHubIssue;
 	
+	private IHttpManagerSpi  httpManager;
+	private IHttpClient      httpClient;
+	
 	@Override
 	public void initialise(@NotNull IFramework framework, @NotNull List<IManager> allManagers,
 			@NotNull List<IManager> activeManagers, @NotNull GalasaTest galasaTest) throws ManagerException {
 		super.initialise(framework, allManagers, activeManagers, galasaTest);
-		logger.info("In the GitHubIssueManager");
-		// Check to see if GitHubIssue annotation is present in the test class
-		// If it is, need to activate
+		
+		// Check to see if GitHubIssue annotation is present in the test class or on any of the test methods
+		// If it is, activate this manager
 		
 		if (!required) {
```

Comment:
not sure we need this state being tested.  Once we exit initialise we are (by implication) required